nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/OAuth2Object.java

```diff
@@ -399,23 +485,20 @@ protected NuxeoOAuth2ServiceProvider getProvider(String providerId) {
         return (NuxeoOAuth2ServiceProvider) provider;
     }
 
+    /**
+     * @deprecated since 11.1. Use {@link OAuth2TokenService#getTokens()} instead.
+     */
+    @Deprecated
     protected List<NuxeoOAuth2Token> getTokens() {
-        return getTokens((String) null);
+        return Framework.getService(OAuth2TokenService.class).getTokens();
     }
 
+    /**
+     * @deprecated since 11.1. Use {@link OAuth2TokenService#getTokens(String)} instead.
+     */
+    @Deprecated
```

Comment:
Now, `getClientUserTokens` and `getProviderUserTokens` should use the service instead of this deprecated method.

Reply:
it's already done, will be pushed.